contracts/governance/extensions/GovernorSuperQuorum.sol

```diff
@@ -0,0 +1,49 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.20;
+
+import {Governor} from "../Governor.sol";
+import {IGovernorCounting} from "./IGovernorCounting.sol";
+import {SafeCast} from "../../utils/math/SafeCast.sol";
+import {Checkpoints} from "../../utils/structs/Checkpoints.sol";
+
+/**
+ * @dev Extension of {Governor} with a super quorum. Proposals that meet the super quorum can be executed
+ * earlier than the proposal deadline. Counting modules that want to use this extension must implement
+ * {IGovernorCounting}.
+ *
+ * NOTE: It's up to developers to implement `superQuorum` and validate it against `quorum`.
```

Comment:
I'd move this note to the `superQuorum` function and make it a bit clear about what to expect. Right now the note does not enable the reader to take a decision.